05/DB.test.js

```diff
@@ -1 +1,127 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('DB', () => {
+  describe('insert', async () => {
+    it('Should throw exception when id not a number', async () => {
+      expect.assertions(1);
+      const id = 'pwd';
+      const data = {
+        id: id,
+      };
+
+      const db = new DB();
+
+      try {
+        await db.insert(data);
+      } catch (err) {
+        expect(err).toBe('ID can be only number!');
+      }
+    });
+
+    it('Should throw exception when id is duplicated', async () => {
+      expect.assertions(1);
+      const id = 3;
+      const data = {
+        id: id,
+      };
+      const db = new DB();
+      db._rows.push(data);
```

Comment:
Użyłbym 2x `insert` - takie dodawanie bezpośrednio poza zdefiniowane metody może powodować, że kod nie będzie działał jak należy. Mamy określoną funkcję, która dodaje elementy to powinniśmy z niej skorzystać ;)